client/ionic/src/pages/About.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { IonContent, IonPage, IonSlides, IonSlide, IonCard, IonCardContent, IonCardHeader, IonCardSubtitle, IonCardTitle, IonButton, IonImg } from '@ionic/react';
```

Comment:
do we want to try to keep our imports in alphabetical order? will #82 enforce that?

Reply:
@advayDev1

Reply:
I don't believe gts enforces this. I wouldn't say it's necessary though, and we should probably stick with gts's opinionated style.

Reply:
GTS provides an option that is off by default (via tslint), IMO: submit as is, and we can just run `npm run fix` if we decide to later.